README.md

```diff
@@ -0,0 +1,87 @@
+Handling Asynchronous Processes with Callbacks
+===============================================
+
+## Overview
+
+In this lesson we will learn how to manage a common challenge that one encounters when architecting Node applications: namely, what do you do when you need a part of your program to run *now* and then another part of it to run *later*? This situation, known as "asychronous processing" (or just "async"), is a key Node programming skill. By the end of this lesson you'll be able to:
```

Comment:
Could we tighten this up? I really like how you present the problem ("What do you do when you need a part of your program..."), and I wonder if we can really zero in to focus on that question.